tools/run_test_case.py

```diff
@@ -36,6 +36,8 @@ def run_alpha_test( fim_run_dir, version, test_id, magnitude,
     # make bool
     calibrated = bool( calibrated )
 
+    print(f"run_alpha_test: test_id = {test_id}")
```

Comment:
This might mess with the progress bar that's set up in `synthesize_test_cases.py`